qlv.py

```diff
@@ -353,6 +363,16 @@ def _insert_step_ignore_diff(self, parent, insert_index, step, tags):
                 values = ("-", "-", str(step))
             )
 
+    def _insert_interrupt_row(self, parent, insert_index, step, tags):
```

Comment:
Maybe rename `step` to `interrupt` (because `inst` in `_insert_instruction_row`)?